packages/components/src/markdown/Markdown.js

```diff
@@ -23,18 +24,17 @@ const Markdown = ({ as: Element, children, className, allowList, blockList }) =>
 
   const parser = (nodes) => {
     const parsed = reader.parse(nodes);
-
     if (allowList || blockList) {
       const toExclude = allowList ? difference(NODE_TYPE_LIST, allowList) : blockList;
-
       return stripNodes({ parsed, blockList: toExclude });
     }
     return parsed;
   };
 
   const createMarkup = () => {
     const parsed = parser(children);
-    return writer.render(parsed);
+    const markup = writer.render(parsed);
+    return linkTargetBlank ? markup.replace(/<a href="/g, `<a target="_blank" href="`) : markup;
```

Comment:
then use the config here

```suggestion
    return linkTargetBlank ? markup.replace(/<a href="/g, `<a target="${config.link.target}" href="`) : markup;
```